xml/System.Threading/Thread.xml

```diff
@@ -1826,8 +1826,8 @@ Main thread: ThreadProc.Join has returned.  Press Enter to end program.
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Gets the cached processor ID used as a hint for which per-core stack to access. It is periodically refreshed to trail the actual thread core affinity.</summary>
+        <returns>An integer representing the cached processor ID.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

This value is cached and periodically refreshed. It may be stale by the time it's used: consuming code must not rely on its accuracy for correctness.

          ]]></format>
        </remarks>
```